tools/profiler.md

```diff
@@ -24,24 +22,18 @@ Regarding latency, it is a whole topic in itself, but remember that we are talki
 
 ## Prerequisites
 
-Starting from Unity 2021.1, the Unity profiler can extend its features with Netcode for GameObjects [NetCode]. Netcode enables you to profile your game in the same tool you are already using for CPU, GPU, and memory.
-
-To enable profiling support for Netcode, install the `com.unity.multiplayer.tools` package from the [Unity Package Manager](../tools/install-tools.md). Then, you should see two additional modules in your profiler (**Window** > **Analysis** > **Profiler**):
-* Messages - Network messages sent to the transport
-* Objects - Activity related to, and grouped by, network objects
```

Comment:
I think this section maybe shouldn't have been cut, as it defines the modules that are referenced in the later paragraph which says:
> you may need to close and reopen the profiler window for the modules to appear

Reply:
Makes sense. Rereading it again, I realize how I misinterpreted that section from when I originally wrote it. Adding it back.